todolist/src/components/TodoContainer.js

```diff
@@ -3,84 +3,164 @@ import TodosList from "./TodosList";
 import Header from "./Header"
 import InputTodo from "./InputTodo"
 import { v4 as uuidv4 } from "uuid";
+// class TodoContainer extends React.Component {
+//     state = {
```

Comment:
- [ ] [OPTIONAL] The line highlighted will be better represented in an ES6 syntax format. Here is what I mean : const TodoContainer () => {}
- This is good practice as it improves readability, Improves code organisation,  and better performance